administrator/language/de-DE/com_actionlogs.ini

```diff
@@ -23,12 +24,18 @@ COM_ACTIONLOGS_EXTENSION="Erweiterung"
 COM_ACTIONLOGS_EXTENSION_ASC="Erweiterung absteigend"
 COM_ACTIONLOGS_EXTENSION_DESC="Erweiterung aufsteigend"
 COM_ACTIONLOGS_ERROR_COULD_NOT_EXPORT_DATA="Daten konnten nicht exportiert werden."
+COM_ACTIONLOGS_FIELD_VALUE_DELETE="DELETE"
+COM_ACTIONLOGS_FIELD_VALUE_GET="GET"
+COM_ACTIONLOGS_FIELD_VALUE_PATCH="PATCH"
+COM_ACTIONLOGS_FIELD_VALUE_POST="POST"
+COM_ACTIONLOGS_FIELD_VALUE_PUT="PUT"
 COM_ACTIONLOGS_IP_ADDRESS="IP-Adresse"
 COM_ACTIONLOGS_IP_ADDRESS_ASC="IP-Adresse aufsteigend"
 COM_ACTIONLOGS_IP_ADDRESS_DESC="IP-Adresse absteigend"
 COM_ACTIONLOGS_IP_INVALID="ungültige IP-Adresse"
 COM_ACTIONLOGS_IP_LOGGING_LABEL="IP-Protokollierung"
 COM_ACTIONLOGS_LOG_EXTENSIONS_LABEL="Ereignisprotokollierung"
+COM_ACTIONLOGS_LOG_VERBS_LABEL="Zu protokollierende Verben"
```

Comment:
```
Zu protokollierende API Verben
```

?

Reply:
Sind das nur API Verben?
War im englischen halt nur "Verbs to log"

Reply:
Ja ist nur bezüglich API relevant.